.github/workflows/build.yaml

```
@@ -4,6 +4,9 @@ on:
   pull_request:
     branches:
       - '*'
+  paths-ignore:
+    - 'docs/**'
+    - '**.md'
```

This would allow prepare release PRs to be ignored:
```suggestion
    - '**.md'
    - 'script/gcg/**'
```